src/m1_facilitator/manifest/Manifest.ts

```diff
@@ -78,7 +78,10 @@ interface ManifestInfo {
 
 /** Enum of architecture layouts which facilitator supports. */
 export enum ArchitectureLayout {
-  MOSAIC1 = 'MOSAIC1',
+  // For supporting anchor contract in mosaic-1.
+  MOSAIC1_0_14 = 'MOSAIC1_0_14',
```

Comment:
Mosaic_v0.14 can work on both gen0 and gen1 testnets.

Reply:
Why not `MOSAIC_0.14_GEN_0`

Reply:
Changed it to :
```
export enum ArchitectureLayout {
  MOSAIC_0_14_GEN_0 = 'MOSAIC_0_14_GEN_0',
  MOSAIC_0_14_GEN_1 = 'MOSAIC_0_14_GEN_1',
}

```